packages/auth0_server_python/src/auth0_server_python/auth_server/server_client.py

```diff
@@ -42,30 +42,33 @@
 
 # Generic type for store options
 TStoreOptions = TypeVar('TStoreOptions')
+INTERNAL_AUTHORIZE_PARAMS = ["client_id", "redirect_uri", "response_type",
+                             "code_challenge", "code_challenge_method", "state", "nonce"]
+
```

Comment:
☝️  1/2 relevant changes on this file. We make sure sensitive internal params are not overrode by user provided params on request. These are significant for retrieving and validating transaction and transaction state, so can't be touched.